### PR TITLE
New envSlice function

### PR DESCRIPTION
**Pull request by potatop**
I added envSlice function so you can do range templates. For example
`ENV_TENANTS=bam:ham:pam`
and template 
```
{{range envSlice "ENV_TENANTS"}}
  path: {{.}}/somepath
{{end}}
```
would produce
```
  path: bam/somepath
  path: ham/somepath
  path: pam/somepath
```

### CODE REVIEW

File: envtemplate.go [hunk 1]
```diff
@@ -196,6 +203,14 @@ func (r *runner) envOrDefault(key, defValue string) string {
 	return value
 }
 
+func (r *runner) envSlice(key string) ([]string, error) {
```
Comment (9len) [inline envtemplate.go]: let's call it `envSplit` instead, and make it take the delimiter as an explicit argument.

**Review by 9len — COMMENTED** [review]
Thanks for the contribution! After you address the feedback, can you rebase your changes as a single commit? Then I will merge it to our internal monorepo and push it back out.

File: envtemplate.go [hunk 2]
```diff
@@ -46,6 +47,10 @@ Two functions are made avaiable to the templates:
 with a default value, which can reference other environment variables:
     {{print "{{envOrDefault \"TBN_HOME\" \"~/$TBN_WORKSPACE/tbn\"}}"}}
 
+{{ul "envSplit"}}: used to specify a required environment variable
+whose value is a list separated by ":" character:
```
Comment (9len) [inline envtemplate.go]: doc needs an update still.

**Review by 9len — COMMENTED** [review]
looks great! will get it merged down on our side. thanks!

**Comment by potatop** [comment]
Thanks. Glads you find this potentially useful.

**Pull request closed without merging**